generator/src/test/java/com/linkedin/pegasus/generator/TestDataSchemaParser.java

```diff
@@ -92,6 +102,57 @@ public void testParseFromJarFile(String pegasusFilename, String[] expectedSchema
     parseResult.getSchemaAndLocations().values().forEach(loc -> assertEquals(loc.getSourceFile().getAbsolutePath(), jarFile));
   }
 
+  @DataProvider(name = "entityRelationshipInputFiles")
+  private Object[][] createResolverWithExtensionDirs()
+  {
+    return new Object[][]
+        {
+            {
+                new String[]{
+                    "extensions/Bar.pdl",
+                    "extensions/FooExtension.pdl",
+                    "pegasus/Foo.pdl"
```

Comment:
Do you think we should also add another test case, a schema under a third path: like "legacyPegasusSchema". The schema under that directory will not be resolved.

Reply:
sure